tagging/images_hierarchy.py

```diff
@@ -0,0 +1,54 @@
+# Copyright (c) Jupyter Development Team.
+# Distributed under the terms of the Modified BSD License.
+from dataclasses import dataclass, field
+from typing import Optional, List
+from taggers import TaggerInterface, \
+    SHATagger, \
+    PythonVersionTagger, \
+    JupyterNotebookVersionTagger, JupyterLabVersionTagger, JupyterHubVersionTagger, \
+    RVersionTagger, TensorflowVersionTagger, JuliaVersionTagger, \
+    SparkVersionTagger, HadoopVersionTagger, JavaVersionTagger
```

Comment:
I haven't yet checked that these values are correct, just added some copy-paste code to make it look like the final version would look like.

I will check the values.

Reply:
Should work now